src/fluree/db/ledger/json_ld.cljc

```diff
@@ -286,30 +319,65 @@
       (->> (conn-proto/-nameservices conn)
            (some #(ns-proto/-alias % db-alias)))))
 
-(defn load
-  [conn db-alias]
+(defn alias-from-address
+  [address]
+  (when-let [path (->> address
+                       (re-matches #"^fluree:[^:]+://(.*)$")
+                       (second))]
+    (if (str/ends-with? path "/main/head")
```

Comment:
I think a TODO comment here could be useful to let us know we have to tighten this up later.